src/components/visualizations/utils/test/drilldownEventing.spec.tsx

```diff
@@ -1298,4 +1302,85 @@ describe("Drilldown Eventing", () => {
             expect(drillEventFunction).toHaveBeenCalledTimes(1);
         });
     });
+
+    describe("Drilling in Geo chart", () => {
+        const defaultDrillConfig = { afm, onFiredDrillEvent: () => true };
+        const target: any = { dispatchEvent: jest.fn() };
+
+        it("should call onDrill", () => {
+            const onDrill = jest.fn();
+            const drillConfig = { ...defaultDrillConfig, onDrill };
+            const drillableProps: IDrillableProps = {
+                drilldown: true,
```

Comment:
this is irrelevant for "handleGeoPushpinDrillEvent" logic, only presence of drillIntersection makes sense

Reply:
removed